app/config/config.yml

```diff
@@ -7,6 +7,7 @@ imports:
 # http://symfony.com/doc/current/best_practices/configuration.html#application-related-configuration
 parameters:
     locale: en
+    swp_updater.version.class: SWP\UpdaterBundle\Version\Version
```

Comment:
this is an example Version class, I just put it here so the UpdaterBundle can work for me.

Do you mean AppBundle or UpdaterBundle ?